docs/tests/scenarios/developers.md

```diff
@@ -0,0 +1,94 @@
+# Developer Scenarios
+
+### General usage of Graasp Desktop and opening Dev tools
+
+**Summary**: A new developer integrates the team and need to understand the app and how he can use it to further develop new features. He first checks whether the Developer Tools are available in the app. He opens Graasp Desktop and visits the software menu. He finds **Toggle Dev Tools** in the menu and enables it. The Dev Tools panel is opened and allows the developer to have a better understanding of the app for debugging. He also visits the **Settings** tab and turn the **Developer Mode** on. He then visits the **Developer** tab. He discovers the JSON database, and eventually checks the mapping of the data to the spaces by going back-and-forth to the spaces and the developer tab.
+
+**Steps:**
+
+1. Open Graasp Desktop
+2. Click on **File**
+3. Click on **Edit**
+4. Click on **View**
+5. Click on **Toggle Developer Tools**
+   <br/><br/>
+6. Click on **Settings**
+7. Enable **Developer Mode**
+8. Click on **Developer**
+9. [play with database]
+
+---
+
+### Testing database changes
+
+**Summary**: To test a new feature, the developer needs to change a space content. Instead of changing it online and synching it in Graasp desktop, he turns the “Developer mode” on, and changes the database in the **Developer** tab. He checks that his changes are reported to the spaces.
+
+**Steps:**
+
+1. Open Graasp Desktop
+2. Click on **Settings**
+3. Enable **Developer Mode**
+4. Click on **Developer**
+5. Change X in X
```

Comment:
We can be specific here. Doesn't need to cover everything, but the most important changes.